js/src/panes/pane.map.js

```diff
@@ -256,18 +256,19 @@ Wu.MapPane = Wu.Pane.extend({
     },
 
     _onMove : function () {
```

Comment:
Firing `projectChanged` doesn't necessarily update project name or slug, does it? I mean, it shouldn't. It's just an event telling the system that project has changed (could be position, could be name, could be settings). So I'm very surprised if firing `projectChanged` sends a server request with project name. 

If so, this has to be fixed, because that's just wrong logic.